File: Sheet 6 (Arrays 1)/B.cpp
#include <iostream>

using namespace std;
int main()
{
    int lev;
    cin >> lev;
    int n , m ;
    bool ck[200] = {0};
    cin >> n ;
    for (int i =0 ;i <n ; i ++){
        int c;
        cin >> c;
        ck[c]=1;
    }
    cin >> m ;
    for (int i =0 ;i <m ; i ++){
        int c;
        cin >> c;
        ck[c]=1;
    }
    for (int  i = 1 ; i <= lev ;  i++){
        if (ck[i] == 0){
            cout <<"Oh, my keyboard!";
            return 0;
        }
    }
    cout << "I become the guy.";
    
    
    return 0;
}

File: Sheet 6 (Arrays 1)/B.py
n = int(input())
a = input()
l1 = a.split()
b = input()
l2 = b.split()
lev = [0]*(n+1)
ck = True
for i in range(1, len(l1)):
    l1[i] = int(l1[i])
    lev[l1[i]] = 1
for i in range(1, len(l2)):
    l2[i] = int(l2[i])
    lev[l2[i]] = 1
for i in range(1 , n+1):
    if lev[i] == 0:
        ck = False
        break
if ck:
    print("I become the guy.")
else:
    print("Oh, my keyboard!")
